product_list.py

```diff
@@ -19,8 +19,4 @@
     {"id":18, "name": "Pizza for One Suprema Frozen Pizza", "department": "frozen", "aisle": "frozen pizza", "price": 12.50},
     {"id":19, "name": "Gluten Free Quinoa Three Cheese & Mushroom Blend", "department": "dry goods pasta", "aisle": "grains rice dried goods", "price": 3.99},
     {"id":20, "name": "Pomegranate Cranberry & Aloe Vera Enrich Drink", "department": "beverages", "aisle": "juice nectars", "price": 4.25}
-] # based on data from Instacart: https://www.instacart.com/datasets/grocery-shopping-2017
-
-print(products)
```

Comment:
Why are we getting rid of this `print` statement? (I'm not saying your change is wrong, I'm genuinely asking)

Reply:
When I imported product_list.py the list would print out every time I ran the program. I don't think we want that (especially without formatting).

I am thinking you will want this as a separate issue just like the line to instruct the clerk to enter 'DONE' when they want to generate the receipt?

Reply:
Given that the whole list was printing every time you ran the program, I think this change seems fine!